packages/strapi-plugin-content-type-builder/admin/src/components/BooleanBox/index.js

```diff
@@ -7,9 +7,22 @@ import Enumeration from './Enumeration';
 import EnumerationWrapper from './EnumerationWrapper';
 import Wrapper from './Wrapper';
 
-const BooleanBox = ({ label, name, onChange, options, value }) => {
+const BooleanBox = ({
+  label,
+  name,
+  onChange,
+  onChangeCallback,
+  options,
+  value,
+}) => {
   const { formatMessage } = useGlobalContext();
 
+  const handleOnchange = e => {
```

Comment:
We usually name `handleChange` to be simple, maybe can you call it the same?

Reply:
Yes, thank you! That's a miss